code/modules/client/preference_setup/general/03_body.dm

```diff
@@ -245,6 +245,10 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 				organ_name = "liver"
 			if(BP_KIDNEYS)
 				organ_name = "kidneys"
+			if(BP_STOMACH)
```

Comment:
This is getting extremely bloaty. At this point this should just use the species internal organs and go from there.